localization/english/wc_traits_l_english.yml

```diff
@@ -506,4 +506,8 @@
 
  trait_creature_drogbar:0 "Drogbar"
  trait_creature_drogbar_desc:1 "The Drogbar are a race created by the Titans by modifying the Trogg. While the they were formerly slaves to the black dragon Neltharion, the Drogbar were freed by Huln Highmountain after the War of the Ancients. They are particularly adept in earth magic, and as such they use rocks for most purposes. With their magic rocks are carved to be used for furniture, containers, and even instruments like drums. They are even masters at Jewelcrafting; in particular the chief shaman of a tribe usually wears a neckless of heave gems."
- trait_creature_drogbar_character_desc:0 "$trait_creature_drogbar_desc$"
\ No newline at end of file
+ trait_creature_drogbar_character_desc:0 "$trait_creature_drogbar_desc$"
+
+ beauty_good_lore:0 "Elegant"
```

Comment:
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/114560463-c67f2b00-9c7d-11eb-9e02-c233ea733c80.png)

</details>

```suggestion
 trait_beauty_good_lore:0 "Elegant"
```